eng/Subsets.props

```diff
@@ -217,6 +217,32 @@
     <ProjectToBuild Include="$(MonoProjectRoot)netcore\System.Private.CoreLib\System.Private.CoreLib.csproj" Category="mono" />
   </ItemGroup>
 
+  <!-- Host sets -->
+  <ItemGroup Condition="$(_subset.Contains('+host.native+'))">
```

Comment:
Do we want the host to be build before libraries, or just libs.tests?

cc: @ViktorHofer @jkoritzinsky there are some discussions about doing this longterm in order to use the host to run libraries tests.

Reply:
Reasonable question, I'll defer. But generally, I think the hosts themselves should probably be moved into the CLR partition -- being in the installer partition seems like more of historical circumstance than a deliberate architecture